src/js/components/dashboard-footer.vue

```diff
@@ -56,21 +56,42 @@
 </template>
 
 <script>
-import ExternalLink from './external-link';
+import ExternalLink from "./external-link";
```

Comment:
this is what you were talking about with the code formatter? i'm fine leaving it for now and I have a note to implement some sort of more consistent code formatting specs on the project anyway..

Reply:
Yep - sounds good!